docs/plasmid/statistics.md

```diff
@@ -85,7 +85,7 @@ The final step is to scatter these increments around your minigame and collect s
 ## Finished!
 And then that's it, your minigame now has statistics support 🎉!
 
-If you need a hand implementing or don't understand something, feel free to [join the Discord](https://nucleoid.xyz/discord/) and speak to me (@Tom_The_Geek#8559) in `#minigame-dev`.
+If you need a hand implementing or don't understand something, feel free to [join the Discord](https://nucleoid.xyz/discord/) and speak to me (@tom_the_geek) in `#minigame-dev`.
```

Comment:
It's probably worth making this a more generic "ask for help in minigame-dev", as Tom hasn't been involved in the project for at least a year or so 😄

Reply:
That's true 😃